deploy/examples/csi-topology/02-operator.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ember-csi-operator
+  namespace: ember-csi
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: ember-csi-operator
+  template:
+    metadata:
+      labels:
+        name: ember-csi-operator
+    spec:
+      serviceAccountName: ember-csi-operator
+      containers:
+        - name: ember-csi-operator
+          image: quay.io/kirankt/ember-csi-operator:v0.0.5
```

Comment:
There seems to be an issue with this version of the operator, it panics - please see attached logs:
[topoerrors.txt](https://github.com/embercsi/ember-csi-operator/files/3112728/topoerrors.txt)

Reply:
The latest commit fixes this bug. Thanks for the thorough review.